resources/runners/qemu.sh

```diff
@@ -17,9 +17,14 @@ do
     shift
 done
 
-grep -q -e "vmx" -e "svm" /proc/cpuinfo
-if [ $? -eq 0 ]; then
-  QEMU="$QEMU -enable-kvm"
+# Check for hardware-virtualization support
+if [ "$(egrep -m 1 '^flags.*(vmx|svm)' /proc/cpuinfo)" ]
+then
+    echo ">>> KVM: ON "
+    export QEMU="qemu-system-x86_64 -M accel=kvm:tcg"
```

Comment:
Don't use `export` nor set explicitly the command to be used, just reassign the `QEMU` environment variable like it was done before. Have you check if the `-enable-kvm` flag actually doesn't work on Windows? If so, have you check the `-M accel=kvm:tc` arguments enable KVM on Linux?